### PR TITLE
Run drush core-cron

### PR DESCRIPTION
**Pull request by nlisgo**

### CODE REVIEW

File: salt/journal-cms/cron.sls [hunk 1]
```diff
@@ -1,10 +1,10 @@
 #
 # cron jobs
 #
-journal-cms-cron-cache-rebuild:
```
Comment (nlisgo) [inline salt/journal-cms/cron.sls]: We no longer need to clear cache regularly as we have disabled cache.

File: salt/journal-cms/cron.sls [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
+journal-cms-cron-drupal:
     cron.present:
-        - identifier: journal-cms-cron-cache-rebuild
```
Comment (giorgiosironi) [inline salt/journal-cms/cron.sls]: to really remove the previous cron, another `cron.absent` needs to be created (temporarily until is applied to all instances) with the same `name`, `identifier`, and `user`:
https://docs.saltstack.com/en/latest/ref/states/all/salt.states.cron.html#salt.states.cron.absent

**Comment by giorgiosironi** [comment]
Added the edit to `cron.absent`. The build has some problem during the creation of the stack, not sure yet of the cause but it has been repeatable. /cc @lsh-0 in case you see something similar in other formulas.

**Comment by giorgiosironi** [comment]
Green in Vagrant.

**Comment by giorgiosironi** [comment]
Error now is 
```
[34.229.164.83] out:           ID: mysql-ppa
[34.229.164.83] out:     Function: cmd.run
[34.229.164.83] out:         Name: apt-key adv --keyserver pgp.mit.edu --recv-keys 5072E1F5
[34.229.164.83] out:       Result: False
[34.229.164.83] out:      Comment: Command "apt-key adv --keyserver pgp.mit.edu --recv-keys 5072E1F5" run
[34.229.164.83] out:      Started: 14:18:25.514948
[34.229.164.83] out:     Duration: 63190.396 ms
[34.229.164.83] out:      Changes:   
[34.229.164.83] out:               ----------
[34.229.164.83] out:               pid:
[34.229.164.83] out:                   22921
[34.229.164.83] out:               retcode:
[34.229.164.83] out:                   2
[34.229.164.83] out:               stderr:
[34.229.164.83] out:                   gpg: requesting key 5072E1F5 from hkp server pgp.mit.edu
[34.229.164.83] out:                   gpg: no valid OpenPGP data found.
[34.229.164.83] out:                   gpg: Total number processed: 0
[34.229.164.83] out:               stdout:
[34.229.164.83] out:                   Executing: gpg --ignore-time-conflict --no-options --no-default-keyring --homedir /tmp/tmp.VoEtI6F0QT --no-auto-check-trustdb --trust-model always --keyring /etc/apt/trusted.gpg --primary-keyring /etc/apt/trusted.gpg --keyring /etc/apt/trusted.gpg.d/fkrull-deadsnakes-python2_7.gpg --keyserver pgp.mit.edu --recv-keys 5072E1F5
[34.229.164.83] out:                   gpgkeys: key 5072E1F5 can't be retrieved
``` 
which at least is more understandable. The Python 2.7 PPA being down.

**Comment by giorgiosironi** [comment]
The key id is correct and is for MySQL. The state is timing out, and even locally it takes tens of seconds.

**Comment by giorgiosironi** [comment]
This need not being blocked though.

**Pull request merged**